packages/sdk-components-react-radix/src/sheet.ws.tsx

```diff
@@ -71,35 +70,135 @@ export const metaSheetContent: WsComponentMeta = {
     { selector: "[data-side=bottom]", label: "Bottom Side" },
     { selector: "[data-side=left]", label: "Left Side" },
   ],
+  presetTokens: {
+    sheetContent: {
+      /**
+       * fixed w-full z-50
+       * grid gap-4 max-w-lg
+       * m-auto
+       * border bg-background p-6 shadow-lg
+       **/
+      styles: [
+        tc.w("full"),
+        tc.z(50),
+        tc.flex(),
+        tc.flex("col"),
+        tc.gap(4),
+        tc.border(),
+        tc.bg("background"),
+        tc.p(6),
+        tc.shadow("lg"),
+        tc.relative(),
+        tc.state(
+          [tc.mr("auto"), tc.maxW("sm"), tc.grow()].flat(),
+          "[data-side=left]"
+        ),
+        tc.state(
+          [tc.ml("auto"), tc.maxW("sm"), tc.grow()].flat(),
+          "[data-side=right]"
+        ),
+        tc.state([tc.mb("auto")].flat(), "[data-side=top]"),
+        tc.state([tc.mt("auto")].flat(), "[data-side=bottom]"),
```

Comment:
Must be tokens, we just don't need this states and don't need bind data-side to side property